packages/console/src/utils/user.ts

```diff
@@ -0,0 +1,13 @@
+import { type User } from '@logto/schemas';
+import { conditional } from '@silverhand/essentials';
+
+export const getUserPrimaryIdentity = (user: User) => {
+  const { name, primaryEmail, primaryPhone, username } = user;
+  return name ?? primaryEmail ?? primaryPhone ?? username ?? undefined;
+};
+
+export const getUserSecondaryIdentity = (user: User) => {
+  const { name, primaryEmail, primaryPhone, username } = user;
+
+  return conditional(name && (primaryEmail ?? primaryPhone ?? username)) ?? undefined;
+};
```

Comment:
this is confusing since it looks like there's no relationship between primary and secondary